mod/wet4/views/default/page/elements/title.php

```diff
@@ -18,7 +18,7 @@
 
 //Nick - if this content is in a group the group title will be the h1, the content title will be h2 as it is a child ofthe group
 if($page_owner instanceof ElggGroup){
-    echo "<h2 property='name' class=\"h1\">{$vars['title']}</h2>";
+    echo "<h2 property='name' class=\"h1 title\">{$vars['title']}</h2>";
 }else{
-    echo "<h1 property='name' {$class}>{$vars['title']}</h1>";
+    echo "<h1 property='name' class='title'{$class}>{$vars['title']}</h1>";
```

Comment:
This looks like either `${class}` should be inside, or this might just be causing a double `class` property